docs/commands/zed.md

```diff
@@ -167,6 +166,25 @@ all adhere to the consistency semantics of the Zed lake.
 > a server instance of `zed` on the same file system and data consistency will
 > be maintained.
 
+### Locating the Lake
+
+At times you may want the Zed CLI tools to access the same lake storage
+used by other tools such as [Zui](https://zui.brimdata.io/). To help
+enable this by default while allowing for separate lake storage when desired,
+`zed` checks each of the following in order to attempt to locate an existing
+lake.
+
+1. The contents of the `-lake` option (if specified)
+2. The contents of the `ZED_LAKE` environment variable (if defined)
+3. A Zed lake service running locally at `http://localhost:9867` (if a socket
+   is listening at that port)
+4. A `zed` subdirectory below a path in the
+   [`XDG_DATA_HOME`](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html)
+   environment variable (if defined)
```

Comment:
Maybe tighten this?
```suggestion
4. `$XDG_DATA_HOME/zed` (if the `XDG_DATA_HOME` environment variable is defined)
```

Reply:
@nwt: This one was also intentionally un-tight because even on Windows the `XDG_DATA_HOME` environment variable may be obeyed. Obviously users have their choice of shell on Windows, and some use `$` to expand. But since the out-of-the-box Command Prompt uses dual `%` to expand environment variables, I was keeping it higher level. Maybe another futile hair-split, though.